docs/fullnode/setup.md

```diff
@@ -203,19 +229,33 @@ A part from those, it also uses:
 - port `9090` to expose the [gRPC](https://grpc.io/) service that allows clients to query the chain state
 - port `1317` to expose the REST APIs service
 
-Most of the validators do not have interest in opening any port. However, it might be beneficial to the whole chain if
-you can open port `26656`. This would allow new nodes to connect to you as a peer, making their sync faster and the
-connection more reliable. For this reason, we suggest you opening that port by using `ufw`:
+While opening any ports are optional, it is beneficial to the whole chain if
+you open port `26656`. This would allow new nodes to connect to you as a peer, making their sync faster and the connection more reliable.
```

Comment:
```suggestion
you open port `26656`. This would allow new nodes to connect to you as a peer, making them sync faster and the connections more reliable.
```